### DISCUSSION TITLE
Join address placeholder

### DISCUSSION THREAD

**Discussion by bobhenl** [plan-player-analytics/Plan#3462] [Q&A] [answered]
Hi, is there a join address placeholder? I cannot find it on the Wiki page, but I don't see reason why it wouldn't exist.

**Comment by AuroraLS3** [comment]
No, but it could be easily added - what would you use it for? :)

**Comment by bobhenl** [comment]
Hi, multiple reasons. I'd like to check from where are players joining due to analysing directly ingame using this placeholder, but mainly I use some join addresses where I'd like to make special behaviour on join, for example I have server connected to minehut proxy (one join address variant) and there doesn't work resource pack on bedrock, so I'd like to show them some special messages or add them to the group but for this I need their join addresses (placeholder) so I can compare it on join against the specified value (specific address). Also it would be nice if join address is in the plan ingame info about the player command

**Reply by AuroraLS3** [reply, marked as answer]
I see I'll convert this to a ticket https://github.com/plan-player-analytics/Plan/issues/3463
